### ISSUE TITLE
Difference between download-licenses and add-third-party goals in Maven License Plugin

### ISSUE THREAD

**Issue by aetius1** [mojohaus/license-maven-plugin#372]
I am experimenting with Maven License Plugin in order to generate the third party list of licenses for my app.At my current findings it looks like the download-licenses goal is downloading all the available licenses to license.xml , i am struggling to understand what value does the add-third-party bring , as it seems like i dont need it, was download-licenses will do the same job ? Perhaps i am missing some context here ? I tried to read the documentation on the mojohaus, but i find it vague and insufficient in clear examples , any help will be appreciated

**Comment by caroso-de** [comment]
From a users point of view: 
add-third-party and, in theory, aggregate-add-third-party (see #407) both have a feature that download-licenses is sorely lacking: configuring license-handling/post-processing from remote urls. 

If you (as quite some projects do) decide against a version range for your maven parents, configuring those aspects in a maven parent ist hardly feasible as you'd have to roll out a new parent  to all of your projects everytime you gain some new information about licenses you found.

With the *-add-third-party goals you can simply update a (quite cumbersome) configuration on a remote text file.

I'd very much like to see both goals unified (since the configuration of download-licenses seems much less cumbersome) #313

So long story short: Imho the *-add-third-party goals are better suited for implementing an automated licence-compliance-check atm.
